main.js

```diff
@@ -19,15 +19,19 @@ function Hoverable() {
 	var win = window;
 
 	function init() {
-		win.document.body.setAttribute('data-o-hoverable--js', '');
+		win.document.documentElement.setAttribute('data-o-hoverable--js', '');
 		touchSupport();
 	}
 
+	function updateClasses(action) {
```

Comment:
Also, if I am not mistaken, removing / adding classes on html and body elements causes two reflows on the whole document. That seems pretty costly especially on mobile devices.

I had a gut feeling that this module probably shouldn't exist (use cases are limited and such dependencies come at a price) but now the evidence is starting to pile up. Do we *really* want to pay the price for that dependency that isn't working properly and causes reflows?